File: devtools/platform-descriptor-json/src/main/resources/codestarts/quarkus/core/examples/resteasy-example/java/src/test/java/{package-name.dir}/{resource.class-name}Test.tpl.qute.java
package {package-name};

import io.quarkus.test.junit.QuarkusTest;
import org.junit.jupiter.api.Test;

import static io.restassured.RestAssured.given;
import static org.hamcrest.CoreMatchers.is;

@QuarkusTest
public class {resource.class-name}Test {

    @Test
    public void testHelloEndpoint() {
        given()
          .when().get("{resource.path}")
          .then()
             .statusCode(200)
             .body(is("{resource.response}"));
    }

}

File: devtools/platform-descriptor-json/src/main/resources/codestarts/quarkus/core/examples/resteasy-example/scala/src/test/scala/{package-name.dir}/{resource.class-name}Test.tpl.qute.scala
package {package-name}

import io.quarkus.test.junit.QuarkusTest
import io.restassured.RestAssured.given
import org.hamcrest.CoreMatchers.`is`
import org.junit.jupiter.api.Test

@QuarkusTest
class {resource.class-name}Test {

    @Test
    def testHelloEndpoint() = {
        given()
          .`when`().get("{resource.path}")
          .then()
             .statusCode(200)
             .body(`is`("{resource.response}"))
    }

}